wgsl/index.bs

```diff
@@ -4463,7 +4463,8 @@ effective-value-type.
     creation|pipeline-creation time=].
 4. [=Override-declarations=] are part of the shader interface, but are not
     bound resources.
-5. [=Storage buffers=] and [=type/storage textures=] cannot be [=statically accessed=]
+5. [=Storage buffers=] with a [=access/read_write=] acces mode and 
```

Comment:
nitpick:  We might allow write-only storage buffers some day, so I personally phrase this as "storage buffers with access mode other than [=access/read=], ..."